### DISCUSSION TITLE
Supprimer des issues du repo GitHub de ma SE

### DISCUSSION THREAD

**Discussion by alanzirek** [betagouv/cap#22] [Besoin]
### Besoin
J'ai besoin de supprimer des issues obsolètes dans le repo GitHub de ma SE mais je n'ai pas les droits pour le faire.
https://mattermost.incubateur.net/betagouv/pl/f9c5s6u8r3gu3f88a1uybbowaw

### Solutions envisageables
Activer l'option dans GitHub : 
https://mattermost.incubateur.net/betagouv/pl/f9c5s6u8r3gu3f88a1uybbowaw

### Informations complémentaires

- [ ] J'avais déjà entendu ce besoin au sein de la communauté beta.gouv
- [X] Je me porte volontaire pour participer à la mise en place de la future solution choisie

**Comment by jdauphant** [comment]
Est-ce que tu peux indiquer pourquoi tu as besoin de supprimer des issues ? 
et pourquoi l'option de "fermer comme non planifier" ne pourrait pas servir ?

**Reply by alanzirek** [reply]
Faire le tri car certaines issues ne sont plus pertinentes, doublons... et ne plus les avoir dans la liste des closed.
en effet "fermer comme non planifier" peut être une alternative si l'option de la suppression n'est pas activée.
